src/oc_erchef/apps/oc_chef_wm/src/chef_wm_status.erl

```diff
@@ -71,12 +71,19 @@ check_health(ServerVersion) ->
     KeyGen = chef_keygen_cache:status_for_json(),
     Indexing = chef_index:status(),
 
-    StatList = [{<<"server_version">>, ServerVersion},
-                {<<"status">>, ?A2B(Status)},
+    StatList1 = [{<<"status">>, ?A2B(Status)},
                 {<<"upstreams">>, {Pings}},
                 {<<"keygen">>, {KeyGen}},
                 {<<"indexing">>, {Indexing}}],
-    {Status, chef_json:encode({StatList})}.
+
+    IncludeVersion = envy:get(oc_chef_wm, include_version_in_status, boolean),
```

Comment:
The pattern we usually follow is pattern match in maybe_include_version(true | false | random, StatList1). This function can decide what the helper.to_boolean currently is.

Reply:
Got it. Thanks!